website/src/components/Changelog/GUI.tsx

```diff
@@ -14,7 +14,12 @@ export default function GUI({ title }: { title: string }) {
     <Entries href={href} arches={arches} title={title}>
       {/* When you cut a release, remove any solved issues from the "known issues" lists over in `client-apps`. This cannot be done when the issue's PR merges. */}
       {/*
-      <Entry version="1.2.2" date={new Date(todo)}>
+      <Entry version="1.2.3" date={new Date(todo)}>
+        <ul className="list-disc space-y-2 pl-4 mb-4">
+        </ul>
+      </Entry>
+      */}
+      <Entry version="1.2.2" date={new Date("2024-08-29")}>
```

Comment:
Should probably add "Fixes an issue where Split DNS didn't work for domain-joined Windows machines"?

Reply:
It's in there but I'll reword it

Reply:
Ah, I always seem to miss these